tests/end_to_end/netsh_test_helper.h

```diff
@@ -20,14 +20,14 @@
 
 DWORD WINAPI
 PreprocessCommand(
-    _In_opt_ HANDLE hModule,
+    _In_opt_ const HANDLE hModule,
```

Comment:
```suggestion
    _In_opt_ HANDLE hModule,
```
const not needed here either